### PR TITLE
fix(demo): card quick return to original position without hangs

### LINKED ISSUE #5975: Card hangs in demo
When proceeding from the Group phase to the Vote phase in the retro demo, the card that was being dragged hangs and takes a while to return to its original position.

hanging-card

### AC

- [ ] When proceeding from the Group to the Vote phase in the demo, cards that were being dragged don't hang for several seconds. Either there's a quick transition as they return to their original position or they're immediately in their original position in the Vote phase.

### PR DESCRIPTION
**Pull request by JimmyLv**
# Description

Fixes/Partially Fixes #5975
When proceeding from the Group phase to the Vote phase in the retro demo, the card that was being dragged hangs and takes a while to return to its original position.

## Demo

https://user-images.githubusercontent.com/4997466/187667074-b3de2dc4-8250-4c30-b332-99bd4bb6b706.mp4

## Testing scenarios

- start the retro demo
- go to the Group phase, note that the grouping isn't finished yet 
- move forward to Vote phase, check the dragging card quick transition and return to the original position

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [x] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

**Comment by JimmyLv** [comment]
Hi @BartoszJarocki , Since Nick is out, could you please help review it?

**Review by BartoszJarocki — APPROVED** [review]
lgtm!

File: packages/client/components/ReflectionGroup/RemoteReflection.tsx [hunk 1]
```diff
@@ -181,12 +182,14 @@ const RemoteReflection = (props: Props) => {
       },
       remoteDrag?.isSpotlight
         ? Times.REFLECTION_SPOTLIGHT_DRAG_STALE_TIMEOUT
+        : localPhase.phaseType === VOTE
```
Comment (Dschoordsch) [inline packages/client/components/ReflectionGroup/RemoteReflection.tsx:185]: Does this also happen in normal retros? Can you talk me through how this happens? I'm wondering if modifying the `RemoteReflection` component just for the demo is the correct thing to do.

Comment (JimmyLv) [reply packages/client/components/ReflectionGroup/RemoteReflection.tsx:185]: @Dschoordsch  It happens at the moment when going from the Group stage to the Vote stage, https://github.com/ParabolInc/parabol/issues/5975 
The participant's RemoteReflection is still in the process of dragging & dropping(not yet), but the meeting facilitator clicks the "Next" button, so it appears on the Vote page that the RemoteReflection still hangs, and will only return to its original position after the animation delay `Times.REFLECTION_DRAG_STALE_TIMEOUT` here, so I set it directly to 0 so it right back to the original position.

**Review by Dschoordsch — APPROVED** [review]

**Pull request merged**